build_depends.repos

```diff
@@ -2,5 +2,9 @@ repositories:
   # TCP version of transport drivers
   transport_drivers:
     type: git
-    url: https://github.com/MapIV/transport_drivers
-    version: boost
+    url: https://github.com/knzo25/transport_drivers
+    version: feat/multicast_support
+  ros2_socketcan:
+    type: git
+    url: https://github.com/knzo25/ros2_socketcan
+    version: feat/continental_fd
```

Comment:
Please be sure to create a PR against [autowarefoundation/ros2_socketcan](https://github.com/autowarefoundation/ros2_socketcan) so we don't have duplicate dependencies across the Autoware stack :pray:

Reply:
They are very unlikely to accept it :laughing: 
There is a shortage of reviewers there

Reply:
Sad :crying_cat_face: